src/main/java/com/amazon/opendistroforelasticsearch/sql/domain/Order.java

```diff
@@ -24,11 +24,17 @@
 	private String nestedPath;
 	private String name;
 	private String type;
+    private boolean isScript;
```

Comment:
I suggest `isForScriptField`

Reply:
It's not technically correct.
The corresponding DSL for scripted ordering is script, and it's not related to field

```
      "_script": {
        "script": {
          "source": "def date_format_2043496359 = DateTimeFormatter.ofPattern('dd-MM-YYYY').withZone(ZoneId.systemDefault()).format(Instant.ofEpochMilli(doc['insert_time'].value.getMillis()));return date_format_2043496359;",
          "lang": "painless"
        },
        "type": "string",
        "order": "desc"
      }
```